### PR TITLE
Rule details mobile fixes

### PR DESCRIPTION
**Pull request by epwinchell**
jira: https://issues.redhat.com/browse/RHCLOUD-14754

Old
Screen Shot 2021-06-22 at 2 41 48 PM

New
Screen Shot 2021-06-22 at 2 38 20 PM

### CODE REVIEW

**Comment by AllenBW** [comment]
@epwinchell npm i n

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.14.6",
+    "@patternfly/patternfly": "^4.96.2",
```
Comment (AllenBW) [inline package.json:52]: if yah can lock down the version n add the lockfile travis will 🤗  yah

Comment (epwinchell) [reply package.json]: @AllenBW I'm not sure what happened with the package.lock file...

**Review by AllenBW — APPROVED** [review]
Awesome! looks like that version of pf does quite ah bitta dep cleanup ❤️‍🔥

**Pull request merged**